web/src/main/java/com/futurewei/alcor/web/entity/dataplane/InternalPortEntityNB.java

```diff
@@ -0,0 +1,20 @@
+package com.futurewei.alcor.web.entity.dataplane;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.futurewei.alcor.web.entity.port.PortEntity;
+
+import java.util.List;
+
+public class InternalPortEntityNB extends PortEntity {
+
+    @JsonProperty("neighbor_info")
+    private List<HostInfoNB> neighborInfo;
+
```

Comment:
Add one more field 

    @JsonProperty("routes")
    private List<RouteEntity> routes;

Reply:
The field "Routers" is gone. Please add it back.